README.md

```diff
@@ -9,4 +9,4 @@
 5. `go version`
 6. `go get github.com/hooligram/hooligram-server`
 7. `cd ~/go/src/github.com/hooligram/hooligram-server`
-8. `go run main.go&`
```

Comment:
How about the step to define environment variables? Do you think we should include that? @cglotr

Reply:
Agreed. Done fadedc7eabeb0307ecea242cd8ef56432e31db7a